src/sentry/tasks/assemble.py

```diff
@@ -53,6 +53,82 @@ class AssembleTask:
     ARTIFACT_BUNDLE = "organization.artifact_bundle"  # Artifact bundle upload
 
 
+AssembleResult = collections.namedtuple("AssembleResult", ["bundle", "temp_file"])
```

Comment:
```suggestion
class AssembleResult(NamedTuple):
    bundle: File
    temp_file: TempFile
```

IMO this is the typing-compatible way to go

Reply:
I agree, I wanted to keep the refactor of the `assemble_file` minimal but we can for sure go this way!

Reply:
(I didn't know named tuples could also be created by extending the `NamedTuple` base class, python typing is lovely 😅)